athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/TypeRowWriter.java

```diff
@@ -50,11 +49,10 @@ public static RowWriterBuilder writeRowTemplate(RowWriterBuilder rowWriterBuilde
                 rowWriterBuilder.withExtractor(field.getName(),
                         (Float8Extractor) (Object context, NullableFloat8Holder value) -> {
                             value.isSet = 1;
-
                             Map<Object, Object> obj = (Map<Object, Object>) context;
                             ArrayList<Object> objValues = (ArrayList)obj.get(field.getName());
 
-                            value.value = Float.parseFloat(objValues.get(0).toString());
+                            value.value = Double.parseDouble(objValues.get(0).toString());
```

Comment:
I see that we are doing objValues.get(0).toString(). What is the actual type of objValues? do we really need to call toString()? for example, if objValues.get(0) is already a double then turning it into a String just to parse it will significantly reduce performance.